Tests/Unit/Encodings/TransferPayloadTests.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import LibMobileCoin
+@testable import MobileCoin
+import XCTest
+
+class TransferPayloadTests: XCTestCase {
+    static var
```

Comment:
```suggestion
```

Reply:
meh, this was an accidental commit - sorry about that.